components/hero/WhoAreWe.tsx

```diff
@@ -0,0 +1,83 @@
+import { BitcoinIcon, EyeOffIcon } from "lucide-react"
+import { GitHubLogoIcon } from "@radix-ui/react-icons"
+import {
+    CalendarDaysIcon,
+    HandRaisedIcon,
+    InboxIcon,
+    TrashIcon,
+    UsersIcon
+} from "@heroicons/react/24/outline"
+
+export default function WhoAreWe() {
```

Comment:
It's delightful how you structured each content into array so they can be easily rendered into components. I see a lot of places where this is been done in your PR. 

We have a content folder where this is suppose to live, as other content which are displayed across the website lives in that folder. Preferably, you should move this and every other content to the content folder to follow that structure.

Reply:
Great suggestion, definitely a better approach

Reply:
@IgboPharaoh  this has been implemented